proxy/README.md

```diff
@@ -110,7 +110,7 @@ The proxy can output log messages on stderr, which are automatically
 handled by systemd and can be viewed with:
 
 ```
-journalctl -u cc-proxy -f
+sudo journalctl -u cc-proxy -f
```

Comment:
Could you prepend with a `$ ` for the prompt? This seems to be missing for this command and the `enable` one above (but later on in the doc the dollar prompt is used).